gensim/similarities/levenshtein.py

```diff
@@ -52,8 +52,7 @@ class LevenshteinSimilarityIndex(TermSimilarityIndex):
         The DAWG nearest-neighbor search index.
     max_distance : int
         The maximum Levenshtein distance of the most similar terms.
-        Keeping this value below 3 has a significant impact on the
-        retrieval performance.
+        Keeping this value below 2 significantly speeds up the retrieval.
```

Comment:
"Below 3" was actually OK :)

"Below 2" means 0 or 1, which is too restrictive.